source/Armament.h

```diff
@@ -59,6 +59,7 @@ class Armament {
 
 	// Access the array of weapon hardpoints.
 	const std::vector<Hardpoint> &Get() const;
+	std::map<const Outfit *, int> GetEquipped() const;
```

Comment:
We can move the method here if it becomes apparent that multiple classes will benefit from having it.
```suggestion
```